src/Management/src/TracingCore/TracingHostBuilderExtensions.cs

```diff
@@ -26,4 +27,15 @@ public static IHostBuilder AddDistributedTracingAspNetCore(this IHostBuilder hos
     /// <param name="action">Customize the <see cref="TracerProviderBuilder" />.</param>
     public static IWebHostBuilder AddDistributedTracingAspNetCore(this IWebHostBuilder hostBuilder, Action<TracerProviderBuilder> action = null)
         => hostBuilder.ConfigureServices((_, services) => services.AddDistributedTracingAspNetCore(action));
+
+    /// <summary>
+    /// Configure distributed tracing via OpenTelemetry with HttpClient and ASP.NET Core Instrumentation along with (optionally) Zipkin and Wavefront exporting.
+    /// </summary>
+    /// <param name="hostBuilder">Your HostBuilder.</param>
+    /// <param name="action">Customize the <see cref="TracerProviderBuilder" />.</param>
+    public static WebApplicationBuilder AddDistributedTracingAspNetCore(this WebApplicationBuilder hostBuilder,  Action<TracerProviderBuilder> action = null)
```

Comment:
Maybe drop the `Core` in `AspNetCore` from these method names?

Reply:
Hm. Maybe, but I think I'd rather tackle it during upcoming package combining or API review efforts

Reply:
Fine by me. There are probably more occurrences, I haven't checked for that. Just thought I'd point it out.